src/global/right-click/heroes/multiverse-shift.js

```diff
@@ -0,0 +1,188 @@
+const assert = require("../../../wrapper/assert-wrapper");
+const locale = require("../../../lib/locale");
+const { Broadcast } = require("../../../lib/broadcast");
+const { Hex } = require("../../../lib/hex");
+const { ObjectNamespace } = require("../../../lib/object-namespace");
+const { UnitPlastic } = require("../../../lib/unit/unit-plastic");
+const { SystemHighlight } = require("../../highlight-on-system-activated");
+const { Spawn } = require("../../../setup/spawn/spawn");
+const {
+    Card,
+    world,
+    globalEvents,
+    Rotator,
+    GameObject,
+} = require("../../../wrapper/api");
+
+const CARD_NAME = "conservator_procyon";
+const PURGE_CONTAINER_NAME = "bag.purge";
+const ACTION_NAME = "*" + locale("ui.menu.multiverse_shift");
+const FRONTIER_TOKEN_NSID = "token:pok/frontier";
+const SHIPS = [
+    "carrier",
+    "cruiser",
+    "destroyer",
+    "dreadnought",
+    "flagship",
+    "war_sun",
+];
```

Comment:
same SHIPS comment if we want to future-proof should new ships get added (unlikely outside of homebrew)